src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,150 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.qr-code.qr :as qr]
+            [clojure.string :as string]
+            [status-im2.constants :as const]
+            [react-native.blur :as blur]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.list-selection :as list-selection]))
+
+(def ^:const profile-tab-id 0)
+(def ^:const wallet-tab-id 1)
```

Comment:
With all the respect to Go, but this is Clojure, and we often don't create constants like `iota` does. You can actually remove the constants vars and use only keywords, it's more idiomatic. For example, just use `(= @selected-tab :tab/profile)`

Reply:
Nice catch, had no clue how tabs actually worked before, the approach you mentioned makes it simpler no doubt.

I did not know that on changing a tab I'd get back anything other than an integer which was supposed to be the ID of the data map we supply.

fixed here : https://github.com/status-im/status-mobile/pull/15528/commits/724e0e1a5964f565049f5a544683cb061fd90d23